pkg/apis/application/v1alpha1/application_types.go

```diff
@@ -64,6 +66,37 @@ type ApplicationStatus struct {
 	ObservedGeneration int64 `json:"observedGeneration,omitempty"`
 }
 
+// Maintainer contains information about an individual or organization that maintains the source code, images, and
+// package for an Application. An Application can have more than one maintainer.
+type Maintainer struct {
+	// Name is the descriptive name of the maintainer.
+	Name string `json:"name,omitempty"`
+
+	// Url could typically be a website address or a mailto: email address.
+	Url string `json:"url,omitempty"`
```

Comment:
In addition to Name and URL there should be a space for an email address. That would round out the common set of information captured here.

Reply:
Done. This is consistent with how helm does things right.

My original intention was to have the email address captured as a `mailto:` URL, but supporting both URL and email for a maintainer sounds reasonable.